docs/understanding-airbyte/airbyte-protocol.md

```diff
@@ -803,6 +804,55 @@ AirbyteErrorTraceMessage:
         - config_error
 ```
 
+## AirbyteControlMessage
+
+An `AirbyteControlMessage` is for connectors to signal to the Airbyte Platform or Orchestrator that an action with a side-effect should be taken. This means that the Orchestrator will likely be altering some stored data about the connector, connection, or sync.
+
+```yaml
+AirbyteControlMessage:
+  type: object
+  additionalProperties: true
+  required:
+    - type
+    - emitted_at
+  properties:
+    type:
+      title: orchestrator type
+      description: "the type of orchestrator message"
+      type: string
+      enum:
+        - CONNECTOR_CONFIG
+    emitted_at:
+      description: "the time in ms that the message was emitted"
+      type: number
+    connectorConfig:
```

Comment:
`updateConnectorConfig` ?

Reply:
I read our message names as nouns, e.g. the content of this message is a "record", "state", etc.  So, this is a connector config, rather than what we are asking the platform to do, "storeState" or "updateConnectorConfig"